jobs/JEVS_GLOBAL_ENS_STATS

```diff
@@ -1,82 +1,145 @@
-#!/bin/ksh
+#!/bin/bash
 #  JOB card cerated for running METPlus-based Global enesmble verifications
 #   By Binbin Zhou, IMSG@EMC/NCEP
 #   Feb 2, 2022
 #
+#   August 14 2024     Ho-Chun Huang    Merged with JEVS_GLOBAL_ENS_CHEM_GRID2OBS_STATS
+#                                       in #!/bin/bash
+#
+#######################################################
 set -x
+##################################################
+### SENDCOM  - Copy Files From TMPDIR to $COMOUT
+### SENDDBN  - Issue DBNet Client Calls 
+### SENDECF  - Flag Events on ecFLOW
+### SENDMAIL - Send email if file is missing
+##################################################
+export SENDCOM=${SENDCOM:-YES}
+export SENDDBN=${SENDDBN:-YES}
+export SENDECF=${SENDECF:-YES}
+export SENDMAIL=${SENDMAIL:-NO}
+
 #######################################################
+# SET SHELL PROCESSING VARIABLES
 #######################################################
+date
+export PS4='$SECONDS + '
 
 export vhr=${vhr:-00}
 export cycle=${cycle:-t${vhr}z}
 
+##########################################################
+# obtain unique LSF id (jobid) and make temp directories
+##########################################################
 export pid=$$
 export DATA=${DATA:-${DATAROOT:?}/${jobid:?}}
-rm -rf $DATA
+if [ "${RUN}" != "chem" ]; then
+    rm -rf $DATA
+fi
 mkdir -p $DATA
 cd $DATA
 
+export envir=${envir:-prod}
 
-# Run setpdy and initialize PDY variables
-setpdy.sh
-. $DATA/PDY
+# Define COMIN/COMOUT variables:
 
-if [ $RUN = atmos ] || [ $RUN = wmo ] || [ $RUN = headline ] ; then
-  export VDATE=${VDATE:-$PDYm2}
-fi
+echo ${COMPATH}
 
+export NET=${NET:-evs}
+export STEP=${STEP:-stats}
+export COMPONENT=${COMPONENT:-global_ens}
+export RUN=${RUN:-atmos}
+export VERIF_CASE=${VERIF_CASE:-grid2obs}
+export MODELNAME=${MODELNAME:-gefs}
+export modsys=${modsys:-gefs}
+export mod_ver=${mod_ver:-${gefs_ver}}
 
-export SENDCOM=${SENDCOM:-YES}
-export SENDDBN=${SENDDBN:-YES}
-export SENDECF=${SENDECF:-YES}
-export SENDMAIL=${SENDMAIL:-NO}
-
-export COMIN=${COMIN:-$(compath.py $envir/com/$NET/$evs_ver)}
-export EVSIN=${EVSIN:-$COMIN/prep/$COMPONENT/$RUN}
-export EVSINwmo=${EVSINwmo:-$COMIN/prep/$COMPONENT/atmos}
-export COMOUT=${COMOUT:-$(compath.py -o $NET/$evs_ver)}/$STEP/$COMPONENT
-mkdir -p $COMOUT
-
-export COMOUTsmall=$COMOUT/$RUN.$VDATE/$MODELNAME/$VERIF_CASE
-export COMOUTfinal=$COMOUT/$MODELNAME.$VDATE
-
-mkdir -p $COMOUTsmall $COMOUTfinal
+# Run setpdy and initialize PDY variables
+if [ "${RUN}" == "chem" ]; then
+    setpdy.sh 8
+    . ./PDY
+else
+    setpdy.sh 
+    . $DATA/PDY 
+fi
 
 export HOMEevs=${HOMEevs:-${PACKAGEROOT}/${NET}.${evs_ver}}
-export EXECevs=$HOMEevs/exec
-export PARMevs=$HOMEevs/parm
-export USHevs=$HOMEevs/ush
-export SCRIPTerf=$HOMEevs/scripts
-export FIXevs=${FIXevs:-$HOMEevs/fix}
-
+export EXECevs=${EXECevs:-${HOMEevs}/exec}
+export PARMevs=${PARMevs:-${HOMEevs}/parm}
+export FIXevs=${FIXevs:-${HOMEevs}/fix}
+
+if [ "${RUN}" == "chem" ]; then
```

Comment:
Remove `USHevs` and `SCRIPTSevs` this section and just do

`export SCRIPTSevs=${SCRIPTSevs:-${HOMEevs}/scripts}`
`export USHevs=${USHevs:-${HOMEevs}/ush}`

These are the definitions per NCO. If anything in the chem part needs to be updated using `USHevs` as `${HOMEevs}/ush` please do so.

Reply:
Ah, I see it is a typo. I only see this in the J-Jobs so let's so through wit the change.

https://github.com/search?q=repo%3ANOAA-EMC%2FEVS%20SCRIPTerf&type=code